Framework/Algorithms/test/IntegrationTest.h

```diff
@@ -452,9 +452,8 @@ class IntegrationTest : public CxxTest::TestSuite {
 
     for (int j = 0; j < 5; ++j) {
       auto &e = space2D->mutableE(j);
-      space2D->setPoints(j, Points(5, LinearGenerator(0, 0.9))); // assign X
-      space2D->setCounts(j,
-                         Counts(5, LinearGenerator(double(j), 2))); // assign Y
+      space2D->setPoints(j, 5, LinearGenerator(0, 0.9));       // assign X
+      space2D->setCounts(j, 5, LinearGenerator(double(j), 2)); // assign Y
```

Comment:
A while ago we decided to remove all C-style casts. Always use `static_cast` instead.